src/kOS/AddOns/KerbalAlarmClock/KACWrapper.cs

```diff
@@ -55,6 +55,17 @@ public class KACWrapper
 
         public static Boolean NeedUpgrade { get; private set; }
 
+        internal static Type GetType(string name)
```

Comment:
This doesn't really need to be `internal`, doing so in the `IRWrapper` makes a little sense because it is consistent with other wrappers, but it could simply be private for `KACWrapper`.  Not a big enough deal to require a change alone, but I'm going to request another change anyways so I thought I might as well comment.